src/sentry/static/sentry/app/components/listView.jsx

```diff
@@ -57,14 +56,14 @@ class ListView extends React.Component {
     columns: PropTypes.array.isRequired,
     errorMessage: PropTypes.string,
     loading: PropTypes.bool.isRequired,
-    orgId: PropTypes.string.isRequired,
     canSelect: PropTypes.bool.isRequired,
     allVisibleSelected: PropTypes.bool.isRequired,
     toggleAll: PropTypes.func.isRequired,
     toggleSingle: PropTypes.func.isRequired,
-    visibleIds: PropTypes.array.isRequired,
```

Comment:
I think we should be able to require this as the initial state defines it (or it should at least) as an empty array, same goes with selectedIds. That would make the code below simpler.